Dockerfile

```diff
@@ -16,7 +16,11 @@ RUN case $TARGETPLATFORM in \
 
 FROM ubuntu:22.04
 
+# install wget & curl
+RUN apt update && apt install -y wget curl
+RUN apt clean && rm -rf /var/lib/apt/lists/*
```

Comment:
We should run update, install and clean in a single command for avoiding unused file remain